rudder-agent/SOURCES/patches/cfengine/0018-do-not-remove-cflags-when-compiling.patch

```diff
@@ -1,10 +0,0 @@
---- a/configure.ac	2015-10-12 11:48:53.451469571 +0200
-+++ b/configure.ac	2015-10-12 11:50:13.759467288 +0200
-@@ -54,7 +54,6 @@
- dnl It is restored later.
- dnl
- ENV_CFLAGS="$CFLAGS"
--CFLAGS=
```

Comment:
This line is no longer in CFEngine 3.6.3's configure.ac and I don't see any other lines in there that reset CFLAGS, so I assume this patch is no longer necessary.